lib/database/user.php

```diff
@@ -1484,53 +1484,62 @@ function getControlPanelUserInfo( $user, &$libraryOut )
 }
 
 function getUserList( $sortBy, $offset, $count, &$dataOut, $requestedBy )
+{
+    return getUserListByPerms( $sortBy, $offset, $count, $dataOut, $requestedBy, NULL );
+}
+
+function getUserListByPerms( $sortBy, $offset, $count, &$dataOut, $requestedBy , &$perms = NULL)
 {
     settype( $offset, 'integer' );
     settype( $count, 'integer' );
 
+    $permsFilter = NULL;
+    if( $perms != NULL)
+    {
+        settype( $perms, 'integer' );
+        if( $perms >= -2 && $perms <= 4 ) // details about the numbers in src/Permissions.php
```

Comment:
could this be 
$perms >= \RA\Permissions::Spam && $perms <= \RA\Permissions::Admin
instead?

Also is there a benefit in allowing people to see Spam accounts? There may be, for other admin at least

Reply:
>  is there a benefit in allowing people to see Spam accounts?

Well, not really sure about that. I'm just curious to see how many of them exist... I'm glad to remove this filter if it makes this PR more acceptable :sweat_smile: